### PR TITLE
Improvement/maps

### PR DESCRIPTION
**Pull request by paturvita**
3 monday items:
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4956373798
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4956374404
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4956375282

TO TEST:
Verify that the map view in whatson events, places and hiking matches the ones in invision and from figma

### CODE REVIEW

File: views/events/index.html [hunk 1]
```diff
@@ -213,7 +213,7 @@
 
 <script>
   mapboxgl.accessToken =
-    'pk.eyJ1IjoiYXNrb2RhbiIsImEiOiJja2drcWR4eW4xYWtuMnN0ZXJpNnJlemk3In0.Dy7UfnJQiXaBYYkN-okLpw';
+    'pk.eyJ1IjoidmlzaXQtaGFsZCIsImEiOiJjbG52dnJxOTEwM3ljMm1vYThhaTJ3ZDRmIn0.D4HoFio_K0-k3aY_QsruCQ';
```
Comment (BartalD) [inline views/events/index.html:216]: Is this a private key?

File: views/events/templates/partials/event-list-list.html [hunk 2]
```diff
@@ -53,21 +53,25 @@ <h2 x-text="new Date(item.dates.date).getDate()  "></h2>
                 </template>
               </div>
             </div>
-            <div class="wrap-list-item">
+            <div class="wrap-list-item event-list-item">
               <div class="list-item-text">
                 <div>
                   <p
-                    x-text="item.dates.eventDatesDays[0].start_time +' - '+ item.dates.eventDatesDays[0].end_time"
```
Comment (BartalD) [inline views/events/templates/partials/event-list-list.html:60]: End time has been removed. Maybe ask [Rasmus](@Horib) if there is some other agreement with VFI/Sendistovan?

File: views/events/templates/partials/event-list-list.html [hunk 3]
```diff
@@ -53,21 +53,25 @@ <h2 x-text="new Date(item.dates.date).getDate()  "></h2>
...
-                  ></p>
+										class="item-text-date"
+                  >
+									<span class="date-day" x-text="item.dates.date.substring(8,10) + '/' + item.dates.date.substring(5, 7)"></span>
```
Comment (BartalD) [inline views/events/templates/partials/event-list-list.html:62]: Invision shows string format in month. E.g.: 24 DEC instead of 24/12

**Review by BartalD — CHANGES_REQUESTED** [review]
## Problems
### Left-panel List
firefox_dW9G38RVc8
firefox_ijwtEx7LPl

- [ ] Date is not in the correct format ([Invision](https://sendistovan.invisionapp.com/console/share/7XFM8NCWDGZ/986217331/play) asks for string in month)
- [ ] Text formatting is off
- [ ] No divider between items on list
- [ ] Scrollbar is visible when list is not scrollable
  - [Rasmus](@Horib) do you have any insights on this? Maybe it should be visible
- [ ] Text-overlay on image should be removed?
- [ ] End-time has been removed (not present in Invision though)
  - [Rasmus](@Horib), do you have any insights here? Maybe VFI asked to have it?

### Map
firefox_0d3bgSuj8C

- [ ] Icons on map are ugly
  - If you zoom in on the viewport they become pretty
- [ ] Map is not zoomable with mousewheel
  - I know this is not in the problem description, but it should be a quick fix?

**Review by HemmingAskodan — CHANGES_REQUESTED** [review]
Rejected for the following reason:

Map's list style overrides the mobile list view. So while the map's list is ok, the mobile list view has gone from OK to NOT OK. The fix here is to separate the stylings, so that the mobile list view gets the old stylings, and the Map's list view gets the new style.

**Comment by HemmingAskodan** [comment]
image

The error above appears when you were in tourism (visitfaroeislands.com) and navigate to whatson. This is because `curl_get_contents` is already declared in the tourism project.
"views\whatson\events.php" did for some reason have this, so I removed it with a commit in this branch.

**Review by HemmingAskodan — APPROVED** [review]

**Review by BartalD — CHANGES_REQUESTED** [review]
image

See my comment above to see the design

Style still doesn't match the design from Sendistovan.
- Date appears to have serifs when they shouldn't
- Months appear to have serifs when they shouldn't
- Alignment of date&month and time seems to be off (might be fixed when fixing bullets above)
- "Place" boldness is not following the design
- Opacity/grayness of fonts is off

File: events.php [hunk 4]
```diff
@@ -90,36 +90,42 @@ function getEvents($path, $translate, $lang, $filterRegion)
...
+            foreach ($data as $idx => $item) {
+                if ($item[0]->type == 'filter') {
+                    $data[$idx] = $this->moveElement($data[$idx], 0, 1);
+                }
 
-                if ($data[$i][0]['type'] == 'filter') {
-                    $data[$i] = $this->moveElement($data[$i], 0, 1);
+                foreach ($item as $subIdx => $subItem) {
+                    if ($subItem->type === 'event' && isset($data[$idx][$subIdx]->dates)) {
+												$dateSplit = $this->get_event_date_split($subItem->dates->date, $lang);
```
Comment (BartalD) [inline events.php:109]: formatting

File: views/events/templates/partials/event-list-list.html [hunk 5]
```diff
@@ -53,21 +53,28 @@ <h2 x-text="new Date(item.dates.date).getDate()  "></h2>
                 </template>
               </div>
             </div>
-            <div class="wrap-list-item">
+            <div class="wrap-list-item event-list-item">
               <div class="list-item-text">
                 <div>
                   <p
-                    x-text="item.dates.eventDatesDays[0].start_time +' - '+ item.dates.eventDatesDays[0].end_time"
-                  ></p>
```
Comment (BartalD) [inline views/events/templates/partials/event-list-list.html:61]: formatting

File: views/events/templates/partials/event-list-list.html [hunk 6]
```diff
@@ -53,21 +53,28 @@ <h2 x-text="new Date(item.dates.date).getDate()  "></h2>
...
+										class="item-text-date"
+                  >
+									<span class="date-day-num" x-text="item.dates.dayNum"></span>
+									<span class="date-month-name" x-text="item.dates.monthName"></span>
+									<span class="date-time" x-text="item.dates.time"></span>
+									</p>
                   <h2 x-text="item.headline"></h2>
-                  <p x-text="item.excerpt"></p>
-                  <!-- <p x-text="item.region_name + (item.region_id  == 6 ? ' {{lang == 'fo' ? translate.capital.fo : translate.capital.en }}': '')"></p> -->
+									<p class="item-text-excerpt" x-text="item.excerpt"></p>
```
Comment (BartalD) [inline views/events/templates/partials/event-list-list.html:67]: formatting

**Review by BartalD — APPROVED** [review]
Some text not showing as design. But we have followed the design code.

**Pull request merged**